app/views/sponsors/toggle_status.js.erb

```diff
@@ -0,0 +1,5 @@
+$("#toggle_text_<%= @sponsor.id %>").text("<%= status_link_text(@sponsor) %>");
```

Comment:
We should use either Jquery or plain Javascript
I would recommend to get yourself familiar with plain javacript first.